src/fsharp/fsi/fsi.fs

```diff
@@ -1490,7 +1496,8 @@ type internal FsiDynamicCompiler
                         packageManagerLines |> List.map (fun line -> directive line.Directive, line.Line)
 
                     try
-                        let result = fsiOptions.DependencyProvider.Resolve(dependencyManager, ".fsx", packageManagerTextLines, reportError m, executionTfm, executionRid, tcConfigB.implicitIncludeDir, "stdin.fsx", "stdin.fsx")
+                        let tfm, rid = tcConfigB.fxResolver.GetTfmAndRid()
```

Comment:
Any reason not have fxResolver on fsiOptions?

Reply:
done thanks